octavia-adoption: perform failover only on amphora LBs (#1372)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_performing-post-adoption-cleanup-of-load-balancers.adoc

$ openstack loadbalancer create --vip-subnet-id public-subnet --name lb-post-adoption --wait
----

. Trigger a failover for all existing amphora load balancers to upgrade the amphorae virtual machines to use the new image and to establish connectivity with the new control plane:
+
----
$ openstack loadbalancer list -f value -c id --provider amphora | \
      xargs -r -n1 -P4 ${BASH_ALIASES[openstack]} loadbalancer failover --wait
----
+
[NOTE]

> OVN load balancers don't support failover as they don't need to be updated during upgrades.

Reviewer: ```suggestion
OVN load balancers do not support failover because they do not need to be updated during upgrades.
```

> OVN load balancers do not support failover because they do not need to be updated during upgrades.

Reviewer: I'm a bit confused by this sentence. During an upgrade, is it optional to update OVN load balancers? And what is the distinction between "update" and "upgrade" in this context?

Author: "updating" an OVN load balancer doesn't make sense, there's nothing to do by the admins.
In this case, there's no distinction between upgrade and update.

Author: I'll try to rephrase it

Author: what about "Failover is not applicable to OVN load balancers. As native components of the OVN infrastructure, they do not require actions during upgrades and updates."

Author: PR updated

Reviewer: Just 1 minor rephrase, and this PR is good to go. I can approve after you commit the suggestion.

> Failover is not applicable to OVN load balancers. As native components of the OVN infrastructure, they do not require actions during upgrades and updates.

Reviewer: ```suggestion
Failover does not apply to OVN load balancers. As native components of the OVN infrastructure, they do not require actions during upgrades and updates.
```
